Sources/Workspace/InitPackage.swift

```diff
@@ -146,6 +148,7 @@ public final class InitPackage {
             }
 
             stream <<< """
+
```

Comment:
This restores a blank line that was accidentally dropped when macro support was added, making the output be the same as in 5.8 and earlier.  It didn't seem worth having a separate PR for just the one line.